.github/workflows/transports-ci.yml

```
@@ -0,0 +1,200 @@
+          node-version: "18"
+
+      # Configure Git for automated commits and tagging
+      # Uses our git-operations script to set up CI user
+      - name: Configure Git
+        working-directory: ci/scripts
+        run: node git-operations.mjs configure
+
+      - name: Determine trigger type and manage versions
+        id: manage_versions
+        working-directory: ci/scripts
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:52: Double quote to prevent globbing and word splitting

(shellcheck)